js/tests/visual/tooltip.html

```diff
@@ -48,14 +56,21 @@ <h1>Tooltip <small>Bootstrap Visual Test</small></h1>
           <button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-bs-placement="left" title="Tooltip with container (selector)" data-bs-container="#customContainer">
             Tooltip with container (selector)
           </button>
+
           <button id="tooltipElement" type="button" class="btn btn-secondary" data-bs-placement="left" title="Tooltip with container (element)">
             Tooltip with container (element)
           </button>
+
           <button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-bs-html="true" title="<em>Tooltip</em> <u>with</u> <b>HTML</b>">
             Tooltip with HTML
           </button>
+
+          <button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="left" title="<img src=1 onerror=alert(123)>">
+            Tooltip with XSS title
+          </button>
+
           <button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-bs-placement="left" title="Tooltip with XSS" data-bs-container="<img src=1 onerror=alert(123)>">
```

Comment:
`container="<img src=1 onerror=alert(123)>"` crashes

Reply:
Yeah, and it's on purpose, to visually see that no XSS happens.

Reply:
Try moving the XSS snippet at the end, to be the last one. It might fix the issue. That being said, the visual tests are only used internally and their purpose might be a little moot in some cases nowadays.

Reply:
I've added a new commit:

```JS
var tooltipXSSContainerElement = document.querySelector('[data-bs-toggle="tooltip-xss-container"]')
try {
  new bootstrap.Tooltip(tooltipXSSContainerElement)
} catch {
  // Should fail
}
```